### PR TITLE
Chrom convert

### PR DESCRIPTION
**Pull request by jfear**
Here is my first attempt at a chromosome convert script. Right now it handles SAM/BAM/GFF/GTF/BED. I started to add BigWig and BigBed (hence my pybedtools PR), but the UCSC tools rely on 'dm6' genome so it is more complicated to change the chromosomes than I originally thought.

I am really liking pytests, read through the docs and started to thinking about making a general fixture plugin.

### CODE REVIEW

File: lcdblib/utils/chrom_convert.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+from lcdblib.logger import logger
+
+
+def arguments():
+    """ Function to pull command line arguments """
+
+    DESCRIPTION = """\
+    Converts between chromosome names.
+
+    In D. melanogaster there are three common varieties of chromosomes names that
```
Comment (daler) [inline lcdblib/utils/chrom_convert.py]: four!

File: lcdblib/utils/chrom_convert.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+    with pysam.AlignmentFile(output, flag_out, header=header) as OUT:
+        for read in curr:
+            OUT.write(read)
+
+def convertFeature(f, mapper):
+    try:
+        f.chrom = mapper[f.chrom]
+        return f
+    except:
```
Comment (daler) [inline lcdblib/utils/chrom_convert.py]: This will convert any exception into a KeyError, might want to drop the try/except.

**Comment by daler** [comment]
PR looks great, but not sure what's going on with the tests. Maybe try rebasing?

**Comment by jfear** [comment]
Yeah, don't know what is going on. Will try to get it working.

**Comment by jfear** [comment]
Very annoying. Everything was working fine, but I blew away my conda env and re-installed everything and am getting import errors. I am guessing this is the issue. Still trying to figure it out.

**Comment by jfear** [comment]
So adding `from . import chrom_convert` to the __init__.py in that folder seems to fix the issue. I never remember when you have to add stuff to the __init__.py and when you can leave it blank.

**Comment by daler** [comment]
I think it was the change to setup.py -- the submodules have to be added manually, or just use find_packages. I hate how finicky python packaging is.

**Comment by jfear** [comment]
Thanks, would have never seen that.

**Pull request merged**